lib/components/narrative/default/itinerary.css

```diff
@@ -4,20 +4,24 @@
   outline: none;
 }
 
+/* If child component is focused, highlight itinerary option */
+.otp .option.default-itin:focus-within {
+  background-color: var(--hover-color);
+}
+
 .otp .option.default-itin {
-  margin-bottom: 16px;
   padding: 10px;
-  border: 2px solid #999;
-  border-radius: 4px
-  background-color: #eeeeee;
+  background-color: white;
 }
 
-.otp .option:hover .default-itin {
-  background-color: var(--hover-color);
+/* Only show border is not expanded */
+.otp .option.default-itin:not(.expanded) {
+  border-bottom: 1px solid grey;
```

Comment:
I think doing a top border would look better, you would have a separator from the narrative-itineraries (right now there is no line between narrative-itineraries and the first itinerary.

Reply:
Or add a top border to the first itinerary shown.

Reply:
I had grown used to not having a line between the "We found..." and the first itinerary, but yea perhaps you're right.